digest/tests/swap_digest.rs

```diff
@@ -52,10 +48,66 @@ impl RootDigest for OtherStruct {
     }
 }
 
+#[derive(DigestMacro)]
```

Comment:
Nit: Would be nice to not have the "Macro" in the name in the end version. I think convention is to name this after the trait is derived.

Reply:
The proc macro doc actually show it like that but happy to remove :)